packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.tsx

```diff
@@ -38,6 +47,7 @@ export class GuxSelectorCard {
   @State()
   private disabled: boolean;
 
+  // Supports the styling of the card border
   @State()
   private checked: boolean;
```

Comment:
Thoughts on renaming this to be `selected` instead of `checked`? It looks like this component's example page has a radio button which got me thinking that the name, `selected`, might be more applicable. Thoughts?